index.md

```diff
@@ -27,6 +27,7 @@ Explore the distributed web
 [Explore](./explore)
 [Blog](./blog/)
 [Protocols](./protocols)
+[Docs](./docs/index)
```

Comment:
With the way index.md works, you can just link to `/docs` and it'll get loaded automagically

```suggestion
[Docs](./docs/)
```

Reply:
Made the changes you asked for! Hopefully you enjoy my emoji usage